### DISCUSSION TITLE
Is it possible to generate a prefetch metadata for those studies archived before enabling this feature?

### DISCUSSION THREAD

**Discussion by luxunxiansheng** [dcm4che/dcm4chee-arc-light#3239] [Q&A] [answered]
Hi, there, 
      Prefetching  data is a great feature.  It works well after we enabled it.  But for those studies archived before, how to generate the json file if possile. 

T.I.A.

L.B.

**Comment by vrindanayak** [comment, marked as answer]
If you are referring to [Prefetch WADO Metadata](https://github.com/dcm4che/dcm4chee-arc-light/wiki/Prefetch-WADO-Metadata) feature, for the studies archived before the export rule was configured, you can simply select single / multiple studies and use `Export Study or Series` or `Export Matching Studies` and select the configured exporter from `Queued Exporter` dropdown. With the latest archive version, the exporter has been extended to also [Prefetch QIDO Responses](https://github.com/dcm4che/dcm4chee-arc-light/wiki/Prefetch-QIDO-Responses)
